src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,77 @@
+package duke.command;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import static duke.command.Command.*;
```

Comment:
Imported classes should always be listed explicitly, as per the java coding standard.